examples/jsm/loaders/EXRLoader.js

```diff
@@ -1756,9 +1756,18 @@ class EXRLoader extends DataTextureLoader {
 
 		}
 
-		function parseInt64( dataView, offset ) {
+		const parseInt64 = function( dataView, offset ) {
 
-			var int = Number( dataView.getBigInt64( offset.value, true ) );
+			let int;
+
+			if ( DataView.prototype.getBigInt64 ) {
```

Comment:
Nit: We usually do `if ( 'getBigInt64' in DataView.prototype )`

Reply:
Sure, will change.